app/src/layouts/organizationsControl/organizationsPopover/organizationsItem/organizationsItemWithPopover/organizationsItemWithPopover.jsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from 'classnames/bind';
+import { useIntl, defineMessages } from 'react-intl';
+import { withTooltip } from 'components/main/tooltips/tooltip';
```

Comment:
I suppose we need to use the tooltip from `componentLibrary` instead of the old from `components/main/tooltips`.
As it has non-changable delay right now, please adjust it to remove the delay or make it configurable.

Reply:
Done.